src/Illuminate/Auth/Notifications/VerifyEmail.php

```diff
@@ -59,7 +59,7 @@ public function toMail($notifiable)
     protected function verificationUrl($notifiable)
     {
         return URL::temporarySignedRoute(
-            'verification.verify', Carbon::now()->addMinutes(60), ['id' => $notifiable->getKey()]
+            'verification.verify', app(Factory::class)->now()->addMinutes(60), ['id' => $notifiable->getKey()]
```

Comment:
This is just going to create a "60 minutes datetime from now", a plain `Carbon::now()->addMinutes(60)` should work fine.